### PR TITLE
Added "bigserial" and "smallserial" to the type mappings

### PR DESCRIPTION
**Pull request by gfoidl**
https://github.com/JasperFx/weasel/pull/87 tried to fix https://github.com/JasperFx/weasel/issues/76, but unfortunately some pieces were missing, as that error still occurs.
With this change that 🐛 should be finally be gone.

(IIRC the change was tested locally and it worked, but now I realized it's not working...)

### CODE REVIEW

File: src/Weasel.Postgresql/Tables/TableColumn.cs [hunk 1]
```diff
@@ -181,12 +181,19 @@ public string FullDeclaration()
...
+
+public class BigSerialValue : ColumnCheck
+{
+    public override string Declaration() => "BIGSERIAL";
+}
+
+public class SmallSerialValue: ColumnCheck
+{
+    public override string Declaration() => "SMALLSERIAL";
 }
```
Comment (gfoidl) [inline src/Weasel.Postgresql/Tables/TableColumn.cs:197]: Are these types actually needed?
If I try to remove them the build and tests pass.

If they're not needed should I remove them?

**Comment by gfoidl** [comment]
On test failed with
```
  Failed Weasel.Postgresql.Tests.ExtensionTests.can_create_extension [194 ms]
  Error Message:
   Shouldly.ShouldAssertException : migration.Difference
    should be
SchemaPatchDifference.None
    but was
SchemaPatchDifference.Create
  Stack Trace:
     at Weasel.Postgresql.Tests.ExtensionTests.can_create_extension() in /home/runner/work/weasel/weasel/src/Weasel.Postgresql.Tests/ExtensionTests.cs:line 28
```
That seems unrelated to the change?

File: src/Weasel.Postgresql/Tables/TableColumn.cs (hunk 1 above)
Comment (oskardudycz) [reply src/Weasel.Postgresql/Tables/TableColumn.cs:197]: Good catch. They're not used in the project, but someone outside of Weasel could use them. Let's keep them for now, I'll revisit them while working on the next major version.

**Review by oskardudycz — APPROVED** [review]
Thank you, @gfoidl!

**Pull request merged**